cmd/file_addplugins.go

```diff
@@ -121,20 +121,20 @@ The plugin files have the following format (JSON or YAML) and are applied in the
 order they are given:
 
 	{ "_format_version": "1.0",
```

Comment:
Forgot to ask earlier: any reason that this is format version 1.0? If no specific reason, then can we put 3.0, to match latest?